kernel/configs/x86_64_kata_kvm_4.19.x

```diff
@@ -1,13 +1,13 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86 4.19.24 Kernel Configuration
+# Linux/x86 4.19.28 Kernel Configuration
 #
 
 #
-# Compiler: gcc (Ubuntu 5.4.0-6ubuntu1~16.04.11) 5.4.0 20160609
+# Compiler: gcc (Ubuntu 7.4.0-1ubuntu1~16.04~ppa1) 7.4.0
```

Comment:
OOI, why are you using a ppa (non-standard) compiler version?

Reply:
My guess - only way to get a recent gcc on bunty...